boss-ws/src/main/java/org/genomebridge/boss/http/resources/AllObjectsResource.java

```diff
@@ -28,8 +24,20 @@ public AllObjectsResource( BossAPI api ) {
 
     @GET
     @Produces("application/json")
-    public Response findObjectsByName( @QueryParam("name") String objectName,
-                                       @HeaderParam(REMOTE_USER_HEADER) String userName ) {
+    @ApiOperation(value = "Querying for Objects by Name",
+                  notes = "Objects can be queried by GETting that name from the /objects URL with the parameter name=objectName.",
+                  response = ObjectDesc.class,
```

Comment:
Technically, we're returning a List<ObjectDesc> so there should also be a responseContainer = "List". See swagger example: https://github.com/swagger-api/swagger-core/blob/master/samples/java-dropwizard/src/main/java/com/wordnik/swagger/sample/resource/PetResource.java